.circleci/config.yml

```diff
@@ -404,6 +404,40 @@ jobs:
           no_output_time: "3m0s"
           <<: *check_scion_connections
 
+      - run:
+          name: NEW -- Check updated configuration matches VPN setup
+          working_directory: ~/project/.circleci
+          command: |
+            # Check configuration on AP
+            echo "Client AP 1405 new VPN configuration"
+            docker-compose exec -T as1405 /bin/bash -c \
+              "apt-get install -y jq > /dev/null 2>&1; jq '"'.BorderRouters."br20-ffaa_0_1405-2".Interfaces'"' /etc/scion/gen/ISD20/ASffaa_0_1405/endhost/topology.json; ip address show tun0"
+            # Check configuration on client
+            echo "Client AS4 new VPN configuration"
+            docker-compose exec -T useras4 /bin/bash -c \
+              "apt-get install -y jq > /dev/null 2>&1; jq '"'.BorderRouters."br20-ffaa_1_4-1".Interfaces'"' /etc/scion/gen/ISD20/ASffaa_1_4/endhost/topology.json; ip address show tun0"
```

Comment:
Is this just debug output, or do you want to keep it?

Reply:
Updated it to produce an exit code that matches the condition, whether the VPN config and the topology match